editions/empty/tiddlywiki.info

```diff
@@ -17,7 +17,7 @@
 		"static": [
 			"--rendertiddler","$:/core/templates/static.template.html","static.html","text/plain",
 			"--rendertiddler","$:/core/templates/alltiddlers.template.html","alltiddlers.html","text/plain",
-			"--rendertiddlers","[!is[system]]","$:/core/templates/static.tiddler.html","static","text/plain"],
-			"--rendertiddler","$:/core/templates/static.template.css","static/static.css","text/plain"
+			"--rendertiddlers","[!is[system]]","$:/core/templates/static.tiddler.html","static","text/plain",
+			"--rendertiddler","$:/core/templates/static.template.css","static/static.css","text/plain"]
```

Comment:
@pmario, I agree, and after a cursory look at some other TiddlyWiki.info files, I think I'll go through all of them and straighten them out